internal/pkg/dynamo/dao/merchant/ddb_tiered_merchant_offer_table.go

```diff
@@ -0,0 +1,43 @@
+package merchant
+
+import(
+	"github.com/blucard/blucard/internal/model/merchant"
+)
+
+// Schema for a table with a single entry containing a map from tier to delegate table name
+const tieredMerchantOfferTableSchema = struct {
```

Comment:
```go
type tieredMerchantOfferTableSchema struct {
	hashKey string `dynamo:"hashKey"`
	mapName string `dynamo:"tieredMerchantOfferMap"`
}
```

although might be cleaner to name the second key `tieredMerchantOfferMap` anyway